internal/pkg/deploy/cloudformation/stack/testdata/workloads/static-site.stack.yml

```diff
@@ -87,18 +87,33 @@ Resources:
         SigningBehavior: always
         SigningProtocol: sigv4
 
+  CloudFrontViewerRequestRewriteFunction:
+    Metadata:
+      'aws:copilot:description': 'CloudFront Function to rewrite viewer request for static site'
+    Type: AWS::CloudFront::Function
+    Properties: 
+      AutoPublish: true
+      FunctionCode: |
+        function handler(e){var i=e.request,n=i.uri;return n.endsWith("/")?i.uri+="index.html":n.includes(".")||(i.uri+="/index.html"),i}
```

Comment:
does this have to be minified 🥺

Reply:
Maybe I'll keep the var name just make the code compact so that we can easily prettify?